spot-price-monitor.py

```diff
@@ -71,6 +106,10 @@ def get_args():
     parser.add_argument('-r', '--region', type=str, default='us-east-1',
                         help='''The region that the cluster is running
                         in (Default: us-east-1)''')
+    parser.add_argument('-o', '--ondemand', action="store_true", default=False,
+                        help='''Will enable ondemand prices''')
+    parser.add_argument('-v', '--verbose', action="store_true", default=False,
+                        help='''Enable verbose output''')
```

Comment:
I wonder if the logging changes are a bit out of scope here.  I'm not sure what the project owners wanted to do about logging, but maybe the debug text should just be removed from the PR and the errors can just be `print`ed ?

Not sure ... perhaps someone from pusher will chime in on this one.

Reply:
I certainly take the point that it might have been better in a different pull request. It is very useful though as it will also show debug messages from the kubernetes and aws api's.